corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -407,6 +437,40 @@ def delete_all_cases(domain=None):
             cursor.execute('SELECT delete_all_cases(%s)', [domain])
 
     @staticmethod
+    def get_all_cases_modified_since(server_modified_on_since=None, chunk_size=500):
```

Comment:
https://github.com/dimagi/commcare-hq/commit/5eab4add42cbbc3ac162ada45a424cad06c3ec95 definitely like this more having done it